pipeline/bicleaner/bicleaner.sh

```diff
@@ -94,6 +87,10 @@ else
     awk -v threshold=${bicleaner_threshold} -F"\t" '{if ($3>threshold) {print $0}}' |
     ${COMPRESSION_CMD} >"${output_prefix}.best.${ARTIFACT_EXT}"
 
+  ${COMPRESSION_CMD} -dc "${output_prefix}.scored.${ARTIFACT_EXT}" |
```

Comment:
I also added saving to artifacts filtered sentences

Reply:
nit: it'd be nice to do these related but not completely related changes in separate commits or separate PRs, so the changes are easier to review